### PR TITLE
added wishlist

### PR DESCRIPTION
**Pull request by kaush499**

### CODE REVIEW

File: connections/mysql_db.js [hunk 1]
```diff
@@ -16,4 +16,10 @@ connection.connect(function(err) {
 });
 
 
+// let query = "select * from users";
+// // let query = "update users set is_verified = 1 where email = 'matajay@gmail.com'";
+// connection.query(query, (e, t)=> {
+//   console.log(t);
+// })
+
```
Comment (pranavpandey1998official) [inline connections/mysql_db.js:24]: why

Comment (kaush499) [reply connections/mysql_db.js:24]: sorry will delete it right away

File: models/wishlist.model.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            if(err) {
+                console.log(err);
+                reject(err);}
+            else resolve(result);
+        });
+    });
+}
+
+const addToWishlist = (body) => {
+    let query = {
```
Comment (pranavpandey1998official) [inline models/wishlist.model.js:22]: check if req.userId = body.userId

File: models/wishlist.model.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        connection.query(query, (err, result) => {
+            // console.log("add" + (JSON.stringify(result)));
+            if(err) reject(err);
+            else resolve();
+        })
+    });
+};
+
+const deleteFromWishlist = (userId, propertId) => {
+    let query = {
```
Comment (pranavpandey1998official) [inline models/wishlist.model.js:37]: check if req.userId = body.userId

File: routes/wishlist.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+const addWishlist = (req, res, next) => {
+    const userId = req.userId;
+    const postUserId = req.body.wishlist.userId;
+    if(userId != postUserId) {
+        return res.status(401).json({
+            error: true,
+            message: "Not authorised to perform this operation"
+        })
+    }
```
Comment (pranavpandey1998official) [inline routes/wishlist.js:30]: add this check to get and delete too

File: models/wishlist.model.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
+const connection = require('../connections/mysql_db');
+
+const getWishlist = (userId) => {
+    let query = {
+        sql: `select * from wishlist as w
```
Comment (pranavpandey1998official) [inline models/wishlist.model.js:5]: in pgSql we could do something like property.* I don't know if this works in mysql too could you try

Comment (kaush499) [reply models/wishlist.model.js:5]: yes it does work

File: routes/wishlist.js (hunk 4 above)
Comment (kaush499) [reply routes/wishlist.js:30]: no its not needed

File: models/wishlist.model.js (hunk 5 above)
Comment (pranavpandey1998official) [reply models/wishlist.model.js:5]: can you change to that , I accidentally merged you pr sorry

File: routes/wishlist.js (hunk 4 above)
Comment (pranavpandey1998official) [reply routes/wishlist.js:30]: yes , sorry about that , I should have looked correctly

**Pull request merged**